sources/packages/backend/apps/api/src/institution-integration/ier-integration/ier12-file-detail.ts

```diff
@@ -0,0 +1,146 @@
+import { DisbursementSchedule, DisbursementValue } from "@sims/sims-db";
+import {
+  BCAG,
+  BCSG,
+  BCSL,
+  BGPD,
+  CSGD,
+  CSGF,
+  CSGP,
+  CSGT,
+  CSLF,
+  SBSD,
+} from "../../constants";
+import {
+  DATE_FORMAT,
+  NUMBER_FILLER,
+  SPACE_FILLER,
+} from "../../cra-integration/cra-integration.models";
+import { round, StringBuilder } from "../../utilities";
+import { IER12RequestFileLine } from "./models/ier12-integration.model";
+
+/**
+ * Record of a IER12 request file.
+ * The documentation about it is available on the document
+ * 'SIMSSFAS - Institution File layouts In Analysis Folder'.
+ */
+export class IER12FileDetail implements IER12RequestFileLine {
+  assessmentId: number;
+  applicationNumber: string;
+  sin: string;
+  studentLastName: string;
+  studentGivenName: string;
+  birthDate: string;
+  programName: string;
+  programDescription: string;
+  credentialType: string;
+  cipCode: number;
+  nocCode: string;
+  sabcCode: string;
+  institutionProgramCode: string;
+  programLength: number;
+  studyStartDate: string;
+  studyEndDate: string;
+  tuitionFees: number;
+  programRelatedCosts: number;
+  mandatoryFees: number;
+  exceptionExpenses: number;
+  totalFundedWeeks: number;
+  courseLoad: number;
+  offeringIntensity: string;
+  disbursementSchedules: DisbursementSchedule[];
+  totalGrants: DisbursementValue[];
+
+  getFixedFormat(): string {
+    this.totalGrants = this.getTotalGrantsFromDisbursementSchedule();
```

Comment:
`this.totalGrants` does not look like private property. Can we please refactor this?